website/docs/r/cloudformation_stack_set.html.markdown

```diff
@@ -85,12 +85,16 @@ resource "aws_iam_role_policy" "AWSCloudFormationStackSetAdministrationRole_Exec
 
 The following arguments are supported:
 
-* `administration_role_arn` - (Required) Amazon Resource Number (ARN) of the IAM Role in the administrator account.
+* `administration_role_arn` - (Optional) Amazon Resource Number (ARN) of the IAM Role in the administrator account. This must be defined when using the `SELF_MANAGED` permission model.
+* `auto_deployment` - (Optional) Nested attribute containing the auto-deployment model for your StackSet. This can only be defined when using the `SERVICE_MANAGED` permission model.
+  * `enabled` - Whether or not auto-deployment is enabled.
+  * `retain_stacks_on_account_removal` - Whether or not to retain stacks when the account is removed.
```

Comment:
Newer linter since this pull request was introduced:

```suggestion
    * `enabled` - (Optional) Whether or not auto-deployment is enabled.
    * `retain_stacks_on_account_removal` - (Optional) Whether or not to retain stacks when the account is removed.
```